server/modules/sslComponentsRepository.js

```diff
@@ -28,7 +28,6 @@ module.exports = function SSLComponentsRepository() {
   };
 
   function loadSSLComponentsFromS3(mainCallback) {
-    // TODO(filip): get rid of async
```

Comment:
Are we now recording these TODO's as tasks elsewhere, or are we letting them go now?